packages/web-components/src/button/button.options.ts

```diff
@@ -10,7 +10,6 @@ export const ButtonAppearance = {
   primary: 'primary',
   outline: 'outline',
   subtle: 'subtle',
-  secondary: 'secondary',
```

Comment:
Secondary no longer exists?

Reply:
It never did - I'm not sure how it got here besides a copy/pasta error or residual from v2